src/connection.h

```diff
@@ -78,6 +78,21 @@ typedef enum {
 #define CONN_TYPE_RDMA "rdma"
 #define CONN_TYPE_MAX 8 /* 8 is enough to be extendable */
 
+static inline const char *getConnectionTypeName(int type_id) {
+    switch (type_id) {
+    case CONN_TYPE_ID_SOCKET:
+        return CONN_TYPE_SOCKET;
+    case CONN_TYPE_ID_UNIX:
+        return CONN_TYPE_UNIX;
+    case CONN_TYPE_ID_TLS:
+        return CONN_TYPE_TLS;
+    case CONN_TYPE_ID_RDMA:
+        return CONN_TYPE_RDMA;
+    default:
+        return "invalid type";
```

Comment:
let's simplify the code a bit further (and remove the \#defines)
```suggestion
static inline const char *getConnectionTypeName(int type_id) {
    switch (type_id) {
    case CONN_TYPE_ID_SOCKET:
        return "tcp";
    case CONN_TYPE_ID_UNIX:
        return "unix";
    case CONN_TYPE_ID_TLS:
        return "tls";
    case CONN_TYPE_ID_RDMA:
        return "rdma";
    default:
        return "invalid type";
```